Sources/PodToBUILD/BuildFile.swift

```diff
@@ -195,7 +195,14 @@ public struct PodBuildFile: SkylarkConvertible {
 
     private static func vendoredFrameworks(withPodspec spec: PodSpec) -> [BazelTarget] {
         let frameworks = spec.attr(\.vendoredFrameworks)
-        return frameworks.isEmpty ? [] : [AppleFrameworkImport(name: "\(spec.moduleName ?? spec.name)_VendoredFrameworks", frameworkImports: frameworks)]
+        return frameworks.map {
```

Comment:
It looks like this might work for a subset of them but is more involved to get working end to end. This was added [here](https://github.com/bazel-xcode/PodToBUILD/pull/234/files#diff-0fd58e621cfd15e65e460905d26ad6c878eb26d7be221329e40ee5bfde4577c5R240)